examples/benchmark.py

```diff
@@ -12,11 +12,11 @@
 from examples.tti.tti_example import run as tti_run
 
 try:
-    from opescibench import Benchmark, Executor, Plotter
+    from opescibench import Benchmark, Executor, RooflinePlotter
 except:
```

Comment:
No need, `opescibench` is a standalone package, at least from a Python point-of-view, so we don't need to be explicit about sub-modules.